core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -1063,6 +1094,38 @@ private ExprValue exprTimeToSec(ExprValue time) {
     return new ExprLongValue(time.timeValue().toSecondOfDay());
   }
 
+  /**
+   * UTC_DATE implementation for ExprValue.
+   *
+   * @param functionProperties FunctionProperties.
+   * @return ExprValue.
+   */
+  private ExprValue exprUtcDate(FunctionProperties functionProperties) {
+    return new ExprDateValue(exprUtcTimeStamp(functionProperties).dateValue());
+  }
+
+  /**
+   * UTC_TIME implementation for ExprValue.
+   *
+   * @param functionProperties FunctionProperties.
+   * @return ExprValue.
+   */
+  private ExprValue exprUtcTime(FunctionProperties functionProperties) {
+    return new ExprTimeValue(exprUtcTimeStamp(functionProperties).timeValue());
+  }
+
+  /**
+   * UTC_TIMESTAMP implementation for ExprValue.
+   *
+   * @param functionProperties FunctionProperties.
+   * @return ExprValue.
+   */
+  private ExprValue exprUtcTimeStamp(FunctionProperties functionProperties) {
+    var zdt = ZonedDateTime.now(functionProperties.getQueryStartClock())
+        .withZoneSameInstant(ZoneId.of("UTC"));
+    return new ExprDatetimeValue(DATE_TIME_FORMATTER_STRICT_WITH_TZ.format(zdt));
```

Comment:
It is possible to avoid conversion `ZonedDateTime -> String -> LocalDateTime` and have  `ZonedDateTime -> LocalDateTime`:
```suggestion
    return new ExprDatetimeValue(zdt.toLocalDateTime());
```